package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
We used `Icons` because, as you know, this was the only icon set available when VSCode didn't even support the feature. I don't have in mind changing this, to be honest. I think that this positions us as the main reference in terms of icons. Even more, people may be used to this shortcut and I wouldn't like to move their cheese.

If we want to use the name in this command, then use it fully or not use it at all, which is indeed what the current message is doing. The thing is that using `VSCode` leads to confusion with the name of the icon set and people may think that there's an error (I did myself) with `VSCode Icon` missing an `s`.